tests/test_hdc_api.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python3
+# SPDX-License-Identifier: MIT
+"""
+Module to test the HDC api basics
+"""
+
+import os
+from benchkit.hdc import *
+
+HOST_FILE = ".gitignore"
+TMP_FILE = "cloned-gitignore"
+TARGET_FILE = "/storage/media/100/local/testing/.gitignore"
+
+device = OpenHarmonyDeviceConnector.query_devices(lambda _: True)[0]

```

Comment:
please enclose the test into a `def main():` and call that with `if __name__ == "__main__":` at the end of the file.